src/fpm_command_line.f90

```diff
@@ -138,7 +138,8 @@ subroutine get_command_line_settings(cmd_settings)
             case default     ; os_type =  "OS Type:     UNKNOWN"
         end select
         version_text = [character(len=80) :: &
-         &  'Version:     0.3.0, alpha',                           &
+         &  'Version:     0.3.0, alpha',                               &
+         &  'PR:          511',                                        &
```

Comment:
This information will become outdated incredibly fast, in case we forget to update or remove it. Also it is a source of merge conflicts for every PR which tries to add it. I recommend against adding it.